crates/pallet-domains/src/lib.rs

```diff
@@ -614,6 +704,24 @@ impl<T: Config> Pallet<T> {
 
         Ok(())
     }
+
+    fn update_tx_range_on_bundle(domain_id: DomainId) {
```

Comment:
I think `do_note_domain_bundle` would be more appropriate since this is not calculating `tx_range` here

Reply:
Renamed without the do_ prefix